### ISSUE TITLE
Brizy Needs Internet connection

### ISSUE THREAD

**Issue by codeclinic** [ThemeFuse/Brizy#1231]
When developing locally and without an active internet connection the Brizy editor will not load beyond the Brizy AJAX loading spinner.

Many developers work on their local laptop/desktop due to it not requiring external resources. I like many developers often find myself doing dev work when I am in locations without a web connection (holidays, journeys, etc). It therefore seems crazy that Brizy will not work unless the dev device hasan active web connection.

I'm currently having to manually edit the HOSTS file on Win machines inorder to have the Brizy Pro editor look locally, which ultimately results in Brizy editor loading up properly. However, this shouldn't be necessary. Brizy either needs to have some sort of timeout facility where it stops searching for https://brizy.io/* and loads the editor. Alternatively this check should be remove or at least moved to not activate until after the editor has loaded.

**Comment by souljoy** [comment]
I agree.  This is something that needs to be considered for those of us that work locally.  I trust the team at Brizy will look in to this.
